metacat-common-server/src/main/java/com/netflix/metacat/common/server/partition/parser/PartitionParserTreeConstants.java

```diff
@@ -14,49 +14,53 @@
 /* Generated By:JavaCC: Do not edit this line. PartitionParserTreeConstants.java Version 6.1_2 */
 package com.netflix.metacat.common.server.partition.parser;
 
-public interface PartitionParserTreeConstants {
-    public int JJTFILTER = 0;
-    public int JJTVOID = 1;
-    public int JJTAND = 2;
-    public int JJTOR = 3;
-    public int JJTNOT = 4;
-    public int JJTBETWEEN = 5;
-    public int JJTIN = 6;
-    public int JJTLIKE = 7;
-    public int JJTCOMPARE = 8;
-    public int JJTGT = 9;
-    public int JJTLT = 10;
-    public int JJTLTE = 11;
-    public int JJTGTE = 12;
-    public int JJTEQ = 13;
-    public int JJTNEQ = 14;
-    public int JJTMATCHES = 15;
-    public int JJTNUM = 16;
-    public int JJTSTRING = 17;
-    public int JJTBOOLEAN = 18;
-    public int JJTVAR = 19;
+public interface PartitionParserTreeConstants
+{
+  public int JJTFILTER = 0;
+  public int JJTVOID = 1;
+  public int JJTAND = 2;
+  public int JJTOR = 3;
+  public int JJTNOT = 4;
+  public int JJTBETWEEN = 5;
+  public int JJTIN = 6;
+  public int JJTLIKE = 7;
+  public int JJTNULL = 8;
```

Comment:
Looks this is the one added. Shall we add a comment to make it easier to understand?

Reply:
This file is generated.